### PR TITLE
v1.2.0

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

Bumps version and updates CHANGELOG for v1.1.2. Also adds mentions of the mailing list.

### Type of Change

- [x] Documentation
- [x] Release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [[Unreleased]]
+
+## [1.1.2] - 2021-11-02
+### Added
+- Support for Python 3.10
```
Comment (ledhed2222) [inline CHANGELOG.md:12]: this should be 1.2.0 because we're adding something

Comment (mvadari) [reply CHANGELOG.md:12]: It still feels like a patch because no code was updated

**Comment by ledhed2222** [comment]
in this case it's still not IMO because we are adding support for a new
version of python. if someone is using 3.10 they will need to pin to our
version >= 1.2.0 and it would be strange to pin to a patch version for that.

**Review by JST5000 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [[Unreleased]]
+
+## [1.1.2] - 2021-11-02
```
Comment (intelliot) [inline CHANGELOG.md:10]:
```suggestion
## [1.2.0] - 2021-11-02
```

**Review by intelliot — APPROVED** [review]

**Pull request merged**